CRD-DTR/HomeOxygenTherapy/R4/resources/Questionnaire-R4-HomeOxygenTherapyFaceToFace.json

```diff
@@ -84,22 +105,36 @@
           ]
         },
         {
-          "linkId": "3.5",
-          "text": "If No and portable O2 is required, describe:  ",
+          "linkId": "3.5.1",
+          "text": "If No, is portable O2 is required?",
```

Comment:
The type of this question should be `boolean` instead of text, since the description portion has been moved to `3.5.2`